### PR TITLE
Documentation about the LOINC code system

### PR DESCRIPTION
**Pull request by rahul1**
This PR is pre-work for #2384 , and documenting how to set up a lab service menu.

### CODE REVIEW

File: packages/docs/docs/careplans/loinc.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+  - LIMS
+---
+
+# LOINC Codes
+
+LOINC (Logical Observation Identifiers Names and Codes) was established with the intention of defining a universal standard for identifying clinical data in electronic reports. The overall scope of LOINC is anything you can **test**, **measure**, or **observe** about a patient.
+
+This guide aims to provide a basic understanding of LOINC codes and how they are applied in practice.
+
+Adopting your clinical data to LOINC codes help:
```
Comment (mattwiller) [inline packages/docs/docs/careplans/loinc.md:24]: Minor phrasing note, but this should probably be either: "Adapting your clinical data to LOINC" or "Adopting LOINC codes for your clinical data"

File: packages/docs/docs/careplans/loinc.md [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+  - diagnostics
+  - loinc
+  - care plans
+  - LIS
+  - LIMS
+---
+
+# LOINC Codes
+
+LOINC (Logical Observation Identifiers Names and Codes) was established with the intention of defining a universal standard for identifying clinical data in electronic reports. The overall scope of LOINC is anything you can **test**, **measure**, or **observe** about a patient.
```
Comment (codyebberson) [inline packages/docs/docs/careplans/loinc.md:20]: Minor: Consider making this first reference to the word "LOINC" into a link to https://loinc.org

**Review by codyebberson — APPROVED** [review]
Nice, looks good.

A couple minor visual recommendations based on https://medplum-www-git-rahul-documentation-loinc-medplum.vercel.app/docs/careplans/loinc

Can be shipped as-is

File: packages/docs/docs/careplans/loinc.md [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+    }
+  ]
+}
+```
+
+</details>
+
+## See Also
+
+- <sup id="ref1">1</sup> https://loinc.org/get-started/scope-of-loinc/
```
Comment (codyebberson) [inline packages/docs/docs/careplans/loinc.md:369]: The `<sup>` looks a little weird.  Maybe add square brackets?  `<sup>[1]</sup>`

The links are not rendering as links.  I think you can fix by surrounding with angle brackets `<https://...>`

image

**Pull request merged**